desertbot/modules/commands/RSS.py

```diff
@@ -0,0 +1,184 @@
+
+        return responses
+
+    @admin("[RSS] Only my admins may follow new RSS feeds!")
+    def _followFeed(self, message: IRCMessage):
+        try:
+            url = message.parameterList[1]
+            name = " ".join(message.parameterList[2:])
+            feed_object = {
+                "url": url,
+                "lastUpdate": datetime.datetime.utcnow() - datetime.timedelta(days=365),
+                "lastTitle": "",
+                "lastLink": "",
+                "lastCheck": datetime.datetime.utcnow() - datetime.timedelta(minutes=10),
+                "suppress": False
+            }
+            self.feeds.append({name: feed_object})
+            self.bot.storage["rss_feeds"] = self.feeds
+            return IRCResponse(ResponseType.Say,
+                               "Successfully followed {} at URL {}".format(name, url),
+                               message.replyTo)
+        except Exception:
+            self.logger.exception("Failed to follow RSS feed - {}".format(message.messageString))
+            return IRCResponse(ResponseType.Say,
+                               "I couldn't quite parse that RSS follow, are you sure you did it right?",
+                               message.replyTo)
+
+    @admin("[RSS] Only my admins may unfollow RSS feeds!")
+    def _unfollowFeed(self, message: IRCMessage):
+        name = " ".join(message.parameterList[1:])
+        if name in self.feeds:
+            del self.feeds[name]
+            self.bot.storage["rss_feeds"] = self.feeds
+            return IRCResponse(ResponseType.Say,
+                               "Sucessfully unfollowed {}".format(name),
+                               message.replyTo)
+        else:
+            return IRCResponse(ResponseType.Say,
+                               "I am not following any feed named {}".format(name),
+                               message.replyTo)
+
+    @admin("[RSS] Only my admins may turn RSS feeds on and off!")
+    def _toggleFeedSuppress(self, message: IRCMessage):
+        name = " ".join(message.parameterList[1:])
+        if name in self.feeds:
+            self.feeds[name]["suppress"] = not self.feeds[name]["suppress"]
+            self.bot.storage["rss_feeds"] = self.feeds
+            return IRCResponse(ResponseType.Say,
+                               "Successfully {}ed {}".format("suppress" if self.feeds[name]["suppress"] else "unsupress", name),
+                               message.replyTo)
+        else:
+            return IRCResponse(ResponseType.Say,
+                               "I am not following any feed named {}".format(name),
+                               message.replyTo)
+
+    def _listFeeds(self, message: IRCMessage):
+        return IRCResponse(ResponseType.Say,
+                           "Currently followed feeds are: {}".format(", ".join(self.feeds.keys())),
+                           message.replyTo)
```

Comment:
you're missing the rss = RSS() at the bottom :P

Reply:
oh I am a smart